lib/block.ml

```diff
@@ -141,14 +143,15 @@ let pp ?syntax ppf b =
   pp_footer ?syntax ppf ()
 
 let labels = [
-  "dir"              , [`Any];
-  "source-tree"      , [`Any];
-  "file"             , [`Any];
-  "part"             , [`Any];
-  "env"              , [`Any];
-  "skip"             , [`None];
-  "non-deterministic", [`None; `Some "command"; `Some "output"];
-  "version"          , [`Any];
+  {s = "dir"              ; t = `Label ; vs = [`Any]};
```

Comment:
I think it would be clearer to write:

```ocaml
let labels = [
  `Label "dir"   , [`Any];
  `Prefix "set-", [`Any];
]
```

What do you think?

Reply:
I love it, would allow to put back the List.assoc and snd. Why I did not think about it before